src/neural/cuda/fp16_kernels.cu

```diff
@@ -434,17 +434,24 @@ void OutputInputTransform(int N, int C, int se_K, T* output, const T* input,
                           const T* b1, const T* w2, const T* b2,
                           cudaStream_t stream) {
   // Each thread processes entire chess board.
-  if (C > kMaxResBlockFusingChannels) {
+  if (use_se == false) {
+    dim3 grid_dim(DivUp(C, kOpInpTransformBlockSize), N, 1);
+    OutputTransform_relu_InputTransform_kernel<half, activation, use_bias,
+                                               use_skip>
+        <<<grid_dim, kOpInpTransformBlockSize, 0, stream>>>(N, C, output, input,
+                                                            (half*)skip, bias);
+  } else if (C > kMaxResBlockFusingChannels) {
     // Use special kernel with reduced register pressure - only works on Ampere,
     // and only for fp16.
     if (C <= kMaxResBlockFusingSeKFp16Ampere) {
       cudaFuncSetAttribute(
           OutputInputTransformKernel_fp16_shmem_board<use_se, activation,
                                                       use_bias, use_skip>,
-          cudaFuncAttributeMaxDynamicSharedMemorySize, 72 * 1024);
+          cudaFuncAttributeMaxDynamicSharedMemorySize,
```

Comment:
This path is only ever called with use_se true now right?

Reply:
correct. Removed the use_se template param.